includes/admin.form.inc

```diff
@@ -234,7 +234,14 @@ function islandora_paged_content_admin_settings_form_gs_ajax_callback(array $for
  */
 function islandora_paged_content_admin_settings_form_djatoka_availible_message($djatoka_url) {
   $file_headers = @get_headers($djatoka_url);
-  $djatoka_availible = strstr($file_headers[0], '200') !== FALSE;
+  $djatoka_availible = FALSE;
+  foreach ($file_headers as $file_header) {
+    if (strstr($file_header, '200') !== FALSE) {
```

Comment:
Also wondering why we use `strstr` here instead of `strpos`. My understanding is it would be more memory efficient as long as we always do `!==` checks.

Reply:
From PHP's [strstr](http://php.net/manual/en/function.strstr.php) page
> Note:
If you only want to determine if a particular needle occurs within haystack, use the faster and less memory intensive function strpos() instead.

Reply:
strpos, 'available', and remove commented code:  done.  New pull request being made with revised code.  Thanks for the quick response.   Moving this conversation to the new pull request.